From 232e13e18e2fb82340dd476ffa7cb6c364fc3399 Mon Sep 17 00:00:00 2001
From: Xie XiuQi <xiexiuqi@huawei.com>
Date: Tue, 29 Nov 2022 23:45:18 +0800
Subject: [PATCH] openEuler: add openEuler/MAINTAINERS for Maintainers &
 Committers
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 232e13e18e2fb82340dd476ffa7cb6c364fc3399
Modified-by-SEL: No


openEuler inclusion
category: doc
bugzilla: https://gitee.com/openeuler/kernel/issues/I63OX8

Add the Maintainer & Committers list for the openEuler/kernel repository
by referring to the MAINTAINERS file of the upstream community.
It is convenient to obtain the list of Maintainers & Committers of the
openEuler/kernel based on the submitted PR or patches.

Only the of the scheduling subsystem is added to openEuler/MAINTAINERS.
This is used as a reference for other modules and subsystems.

Example:
$./scripts/get_maintainer.pl -no-l --no-rolestats
--maintainer-path=openEuler/MAINTAINERS --no-tree --no-fixes
--no-git-blame-signatures --no-git-fallback
0001-sched-fair-Fix-fault-in-reweight_entity.patch

Signed-off-by: Xie XiuQi <xiexiuqi@huawei.com>
(cherry picked from commit 830f093461e080dd58864d28a86f9271863299ba)
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 openEuler/MAINTAINERS | 85 +++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 85 insertions(+)
 create mode 100644 openEuler/MAINTAINERS

diff --git a/openEuler/MAINTAINERS b/openEuler/MAINTAINERS
new file mode 100644
index 000000000000..35b5a8f81fed
--- /dev/null
+++ b/openEuler/MAINTAINERS
@@ -0,0 +1,85 @@
+List of maintainers and committers
+====================================================
+
+Descriptions of section entries and preferred order
+---------------------------------------------------
+
+	M: *Mail* patches to: FullName <address@domain>
+	R: Designated *Reviewer*: FullName <address@domain>
+	   These reviewers should be CCed on patches.
+	L: *Mailing list* that is relevant to this area
+	S: *Status*, one of the following:
+	   Supported:	Someone is actually paid to look after this.
+	   Maintained:	Someone actually looks after it.
+	   Odd Fixes:	It has a maintainer but they don't have time to do
+			much other than throw the odd patch in. See below..
+	   Orphan:	No current maintainer [but maybe you could take the
+			role as you write your new code].
+	   Obsolete:	Old code. Something tagged obsolete generally means
+			it has been replaced by a better system and you
+			should be using that.
+	W: *Web-page* with status/info
+	Q: *Patchwork* web based patch tracking system site
+	B: URI for where to file *bugs*. A web-page with detailed bug
+	   filing info, a direct bug tracker link, or a mailto: URI.
+	C: URI for *chat* protocol, server and channel where developers
+	   usually hang out, for example irc://server/channel.
+	P: Subsystem Profile document for more details submitting
+	   patches to the given subsystem. This is either an in-tree file,
+	   or a URI. See Documentation/maintainer/maintainer-entry-profile.rst
+	   for details.
+	T: *SCM* tree type and location.
+	   Type is one of: git, hg, quilt, stgit, topgit
+	F: *Files* and directories wildcard patterns.
+	   A trailing slash includes all files and subdirectory files.
+	   F:	drivers/net/	all files in and below drivers/net
+	   F:	drivers/net/*	all files in drivers/net, but not below
+	   F:	*/net/*		all files in "any top level directory"/net
+	   One pattern per line.  Multiple F: lines acceptable.
+	X: *Excluded* files and directories that are NOT maintained, same
+	   rules as F:. Files exclusions are tested before file matches.
+	   Can be useful for excluding a specific subdirectory, for instance:
+	   F:	net/
+	   X:	net/ipv6/
+	   matches all files in and below net excluding net/ipv6/
+	N: Files and directories *Regex* patterns.
+	   N:	[^a-z]tegra	all files whose path contains tegra
+	                        (not including files like integrator)
+	   One pattern per line.  Multiple N: lines acceptable.
+	   scripts/get_maintainer.pl has different behavior for files that
+	   match F: pattern and matches of N: patterns.  By default,
+	   get_maintainer will not look at git log history when an F: pattern
+	   match occurs.  When an N: match occurs, git log history is used
+	   to also notify the people that have git commit signatures.
+	K: *Content regex* (perl extended) pattern match in a patch or file.
+	   For instance:
+	   K: of_get_profile
+	      matches patches or files that contain "of_get_profile"
+	   K: \b(printk|pr_(info|err))\b
+	      matches patches or files that contain one or more of the words
+	      printk, pr_info or pr_err
+	   One regex pattern per line.  Multiple K: lines acceptable.
+
+Maintainers List
+----------------
+
+.. note:: When reading this list, please look for the most precise areas
+          first. When adding to this list, please keep the entries in
+          alphabetical order.
+
+SCHEDULER
+M:	zhengzucheng@huawei.com
+S:	Maintained
+F:	include/linux/preempt.h
+F:	include/linux/sched.h
+F:	include/linux/wait.h
+F:	include/uapi/linux/sched.h
+F:	kernel/sched/
+
+THE REST
+M:	xiexiuqi@huawei.com
+M:	zhengzengkai@huawei.com
+L:	kernel@openeuler.org
+S:	Buried alive in reporters
+F:	*
+F:	*/
-- 
2.33.0

